### PR TITLE
Add support to www.ahashpool.com, and a few script fixes

### PR DESCRIPTION
**Pull request by ahashpool**

### CODE REVIEW

File: Pools/ahashpool.ps1 [hunk 1]
```diff
@@ -47,7 +47,7 @@ $ahashpool_Request | Get-Member -MemberType NoteProperty | Select -ExpandPropert
             Host = $ahashpool_Host
             Port = $ahashpool_Port
             User = $Wallet
-            Pass = "ID=$Workername,c=$Passwordcurrency"
+            Pass = "ID=$Workername,c=BTC"
```
Comment (meyerbro) [inline Pools/ahashpool.ps1]: Are you sure this is the best? I believe some people prefer USD for example. It's best to user the parameter no?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,22 @@
-# SniffDog-www.zpool.ca , www.MiningPoolHub.com , www.HashRefinery.com , www.ahashpool.com
```
Comment (meyerbro) [inline README.md]: Why removing the other pools?

**Comment by meyerbro** [comment]
I don't think this PR should be merged as it's removing the support for other pools and also setting BTC as the hardcoded currency to be used as parameter, which is not the best IMHO.

**Comment by ahashpool** [comment]
Pull Request was originally made months ago in September 2017.  The author of this repo did not respond back in Sept.  A lot has changed since then, this pull request is void.

**Pull request closed without merging**